src/main/flight/position.c

```diff
@@ -52,7 +52,7 @@ int16_t calculateEstimatedVario(int32_t baroAlt, const uint32_t dTime) {
 
     int32_t baroVel = 0;
 
-    baroVel = (baroAlt - lastBaroAlt) * 1000000.0f / dTime;
+    baroVel = (lastBaroAlt - baroAlt) * 1000000.0f / dTime;
```

Comment:
This will make going up result in a negative `baroVel`. This sounds unintuitive, and will change the sign compared to how it works now - pretty sure this is wrong.

Reply:
I mentioned this in the comment from 2 days ago, the way it was before was showing up reversed in the osd.

The code in osd.c is very straightforward:

const char directionSymbol = verticalSpeed < 0 ? SYM_ARROW_SOUTH : SYM_ARROW_NORTH;

So I don't really know what's going on, but after swapping Alt and lastAlt, the direction showed up correctly using either the Baro or the GPS (or both when I was mixing the values). This is: arrow pointing up when the quad was gaining altitude, arrow pointing down when quad was descending.

Reply:
Today I've updated the OSD font... and the arrows now work as intended. Arrrrr. Well, at least now all makes sense. I'm amending the code again, sorry for the inconvenience.

Reply:
ROFL